ginmetrics/middleware.go

```diff
@@ -33,6 +33,19 @@ func (m *Monitor) Use(r gin.IRoutes) {
 	})
 }
 
+// Pass endpoints to monitor
+func (m *Monitor) UseWithoutExposingEndpoint(r gin.IRoutes) {
+	m.initGinMetrics()
+	r.Use(m.monitorInterceptor)
+}
+
+// Pass router to expose /metrics
```

Comment:
Change the comments in more detail.

such as:
// Expose used to expose the metrics to specify path. Users can use it to expose metrics
// path to other port different with main server.